### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/blocks/stopwatch.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    border: 0 solid transparent;
+    border-radius: 50%;
+
+    background-color: #a0a0ff;
+
+    animation: rotate 3600s linear infinite normal;
+    transform-origin: bottom center;
+  }
+
+  @keyframes rotate {
```
Comment (yaro-bey) [inline src/blocks/stopwatch.scss:52]: I would move animation in a separate utils file

File: src/blocks/stopwatch.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  &__short-hand {
+    position: absolute;
+
+    height: $shortHandSize;
+    width: 6px;
+    margin-bottom: 90px;
+
+    border: 0 solid transparent;
```
Comment (yaro-bey) [inline src/blocks/stopwatch.scss:43]: same here

Comment (Andriy-Shymkiv) [reply src/blocks/stopwatch.scss:43]: ok , i'll fix it =)

File: src/blocks/stopwatch.scss (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/blocks/stopwatch.scss:52]: idk if it make any sense

File: src/blocks/stopwatch.scss (hunk 2 above)
Comment (mykola-koval) [reply src/blocks/stopwatch.scss:43]: why haven't fixed?

File: src/blocks/stopwatch.scss (hunk 1 above)
Comment (mykola-koval) [reply src/blocks/stopwatch.scss:52]: it would

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**